### ISSUE TITLE
Wrong Contents?

### ISSUE THREAD

**Issue by danchirillo** [sandervanvugt/handson-ansible#1]
The contents in this repo seems to be for a different course ...Ansible in 3 Hours.

**Comment by satishrao84** [comment]
Yes, what's going on @sandervanvugt ?!?! Can you please address this?

**Comment by sandervanvugt** [comment]
Thanks for letting me know - sorry I didn't notice earlier. Will address this shortly.

**Comment by imsanjeevi** [comment]
Please update the content presented in your course - It will be useful for our learning & reference
